src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -31,6 +31,7 @@
 """
 
 
+@staticmethod
```

Comment:
Staticmethod is only relevant for class methods. I don't think it does anything for an independent function.

Reply:
should be been fixed in recent commints, see the outdated tag